ov/hypervisor_cluster_profile.go

```diff
@@ -29,14 +29,37 @@ type HypervisorClusterProfile struct {
 	Path                          string                         `json:"path,omitempty"`                          //"path":"DC1"
 	RefreshState                  string                         `json:"refreshState,omitempty"`                  //"refreshState":"NotRefreshing"
 	ScopesUri                     string                         `json:"scopesUri,omitempty"`                     //"scopesUri":"/rest/scopes/resources/rest/hypervisor-cluster-profiles/4340293c-0701-4773"
-	SharedStorageVolumes          []utils.Nstring                `json:"sharedStorageVolumes,omitempty"`          //"sharedStorageVolumes":"[]"
+	SharedStorageVolumes          []sharedStorageVolumes         `json:"sharedStorageVolumes,omitempty"`          //"sharedStorageVolumes":"[]"
 	State                         string                         `json:"state,omitempty"`                         //"state":"Active"
 	StateReason                   string                         `json:"stateReason,omitempty"`                   //"stateReason":"None"
 	Status                        string                         `json:"status,omitempty"`                        //"status":"OK"
 	Type                          string                         `json:"type,omitempty"`                          //"type":"HypervisorClusterProfileV4"
 	URI                           utils.Nstring                  `json:"uri,omitempty"`                           //"uri":"/rest/hypervisor-cluster-profiles/4340293c-0701-4773-b863-32854b0f7d29"
 }
 
+type sharedStorageVolumes struct {
```

Comment:
Any reason you have kept the struct name in smallcase. Smallcase wont allow it to be exported.

Reply:
updated